www/src/pages/guidelines/design-tokens.js

```diff
@@ -0,0 +1,175 @@
+              </tr>
+            </thead>
+            <tbody>
+              {theme.space.map((space, index) => (
+                <tr key={`tokens-space-${index}`}>
+                  <td>
+                    <code>space[{index}]</code>
+                  </td>
+                  <td>{space.substring(0, space.length - 3)}</td>
+                  <td>{parseFloat(space) * 16}</td>
+                  <td>
+                    {` `}
+                    <Box
+                      key={`${index}-${space}`}
+                      height={space}
+                      width={space}
+                      bg="orange.30"
+                    />
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        </ContentColumn>
+      </Columns>
+    </Section>
+
+    <Section bg="grey.5">
+      <SectionHeading>Shadows &amp; Elevation</SectionHeading>
+      <Columns>
+        <CopyColumn>
+          <p>
+            Component elevation in our UI is depicted using shadows. These
+            tokens are intended to be used with the <code>box-shadow</code> CSS
+            property.
+          </p>
+          <p>
+            There is a lot of definition work to do here, which is why we
+            deliberately chose to omit adding more documentation at this point.
+          </p>
+        </CopyColumn>
+        <ContentColumn
+          css={{
+            alignSelf: `flex-end`,
+            justifyContent: `center`,
+            display: `flex`,
+            flexWrap: `wrap`,
+            overflow: `visible`,
+          }}
+        >
+          {Object.keys(theme.shadows).map((shadow, i) => (
+            <Box
+              bg="white"
+              borderRadius="2"
+              key={`tokens-shadow-${i}`}
+              mx={2}
+              my={8}
+              p={4}
+              width="50%"
+              boxShadow={shadow}
+            >
+              <code>shadows.{shadow}</code>
+            </Box>
+          ))}
+        </ContentColumn>
+      </Columns>
+    </Section>
+
+    <Section>
+      <SectionHeading>Radii</SectionHeading>
+      <Columns>
+        <CopyColumn>
+          <p>
+            Intended for use with the <code>border-radius</code> CSS property.
+            Currently defined in <code>px</code>, which means they do not yet
+            scale when adjusting the root font size.
+          </p>
+        </CopyColumn>
+        <ContentColumn>
+          <table>
+            <thead>
+              <tr>
+                <th>Token</th>
+                <th>px</th>
+                <th>Example</th>
+              </tr>
+            </thead>
+            <tbody>
+              {theme.radii.map((radius, index) => (
+                <tr key={`tokens-radii-${index}`}>
+                  <td>
+                    <code>radii[{index}]</code>
+                  </td>
+                  <td>{radius}</td>
+                  <td>
+                    {` `}
+                    <Box
```

Comment:
Same as the other table: we should add some kind of text alternative for these examples to make the table accessible and valid

Reply:
Added that in a772618 (please also see my comment/question at https://github.com/gatsbyjs/gatsby/pull/14360#discussion_r289421624).